src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -1010,40 +992,43 @@ namespace
         }
     }
 
-    void transform_to_srgb(Image& image)
+    void transform_to_srgb(Image* image)
     {
-        const CanvasProperties& image_props = image.properties();
+        const CanvasProperties& image_props = image->properties();
 
         for (size_t ty = 0; ty < image_props.m_tile_count_y; ++ty)
         {
             for (size_t tx = 0; tx < image_props.m_tile_count_x; ++tx)
-                transform_to_srgb(image.tile(tx, ty));
+                transform_to_srgb(image->tile(tx, ty));
         }
     }
 
-    void write_png_image(
-        const bf::path&         file_path,
-        const Image&            image,
-        ImageAttributes         image_attributes)
+    /*
+     * Default Export Format
+     * OpenEXR   .exr          4-channel   16-bit (half)    Linear  
+     * TIFF      .tiff/.tif    4-channel   32-bit (float)   Linear
+     * RGBE      .hdr          3-channel   32-bit (float)   Linear
+     * BMP       .bmp          4-channel    8-bit (uint8)     sRGB
+     * PNG       .png          4-channel    8-bit (uint8)     sRGB    
+     * JPEG      .jpg/.jpe/    3-channel    8-bit (uint8)     sRGB
+     *           .jpeg/.jif/          
+     *           .jfif/.jfi           
+     */
+    void write_image(
+        const std::string&         filename,
+        std::unique_ptr<Image>&    image,
+        ImageAttributes            image_attributes,
+        PixelFormat                image_output_format = PixelFormat::PixelFormatDefault)
```

Comment:
In fact, as far as I can tell, you're never using that last argument so `PixelFormat::PixelFormatDefault` is truly not needed.

Reply:
No, my expression is not clear.
I want to use it when users can choose what pixelformat they want.

Reply:
If you don't want this, I'll change them in origin function.